### PR TITLE
Add a script for setting GitHub commit/PR status

### PR DESCRIPTION
**Pull request by vpodzime**
We can use this in jenkins to publish the results of jobs.

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @Lex-2008 can review this?

File: build-scripts/set_github_status.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#   $3 - job spec (e.g. "ci/testing-pr/PACKAGES_HUB_x86_64_linux_redhat_7")
+#   $4 - description of the status
+# Env:
+#   $GITHUB_STATUS_TOKEN - token for GitHub authentication
+#
+
+PRs_file="$1"
+state="$2"
+job_spec="$3"
+description="$4"
```
Comment (olehermanse) [inline build-scripts/set_github_status.sh]: missing some characters here? shouldn't it be description and state?

Comment (olehermanse) [reply build-scripts/set_github_status.sh]: Never mind! Just GitHub UI is broken for me.

**Comment by vpodzime** [comment]
@Lex-2008 please re-review.

**Pull request merged**